NetworkChat/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Sockets;
+using System.IO;
+
+namespace NetworkChat
+{
+    class Program
+    {
+        private static async Task Main(string[] args)
+        {
+
+
+            if (args.Length > 2)
```

Comment:
```suggestion
        {
            if (args.Length > 2)
```
После открывающей фигурной скобки, кстати, тоже